mojaloop/chart-admin/templates/deployment.yaml

```diff
@@ -0,0 +1,203 @@
+          {{- end }}
+          ports:
+            - name: http
+              containerPort: {{ .Values.service.internalPort }}
+          {{- if .Values.diagnosticMode.enabled }}
+            - name: debug
+              containerPort: {{ .Values.diagnosticMode.debug.internalPort }}
+          {{- end }}
+          {{- if .Values.readinessProbe.enabled }}
+          readinessProbe:
+            httpGet:
+              path: {{ .Values.readinessProbe.httpGet.path }}
+              port: {{ .Values.service.internalPort }}
+            initialDelaySeconds: {{ .Values.readinessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.readinessProbe.periodSeconds }}
+          {{- end }}
+          {{- if .Values.livenessProbe.enabled }}
+          livenessProbe:
+            httpGet:
+              path: {{ .Values.livenessProbe.httpGet.path }}
+              port: {{ .Values.service.internalPort }}
+            initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
+          {{- end }}
+          volumeMounts:
+            - name: config-volume
+              mountPath: /opt/account-lookup-service/config
+          env:
+            - name: ALS_DATABASE__PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: '{{ include "common.backends.accountdb.secret.name" . }}'
+                  key: '{{ include "common.backends.accountdb.secret.key" . }}'
+            - name: LOG_LEVEL
+              value: {{ .Values.config.log_level | quote}}
+            - name: LOG_FILTER
+              value: {{ .Values.config.log_filter | quote}}
+            - name: CSL_LOG_TRANSPORT
+              value: {{ .Values.config.log_transport | quote}}
+            - name: EVENT_SDK_LOG_FILTER
+              value: {{ .Values.config.event_log_filter | quote }}
+            - name: EVENT_SDK_LOG_METADATA_ONLY
+              value: {{ .Values.config.event_log_metadata_only | quote }}
+            {{- if .Values.sidecar.enabled }}
+            - name: EVENT_SDK_SIDECAR_DISABLED
+              value: "false"
+            - name: EVENT_SDK_SERVER_HOST
+              value: {{ .Values.sidecar.config.event_log_grpc_host | quote }}
+            - name: EVENT_SDK_SERVER_PORT
+              value: {{ .Values.sidecar.config.event_log_grpc_port | quote }}
+            {{- end }}
+            - name: EVENT_SDK_VENDOR_PREFIX
+              value: {{ .Values.config.event_trace_vendor | quote }}
+            - name: EVENT_SDK_TRACESTATE_HEADER_ENABLED
+              value: {{ .Values.config.event_trace_state_enabled | quote }}
+            - name: EVENT_SDK_ASYNC_OVERRIDE_EVENTS
+              value: {{ .Values.config.event_async_override | quote }}
+            - name: EVENT_SDK_TRACEID_PER_VENDOR
+              value: {{ .Values.config.event_traceid_per_vendor | quote }}
+            - name: LIB_RESOURCE_VERSIONS
```

Comment:
Take note that this has changed in v13.1.0 which applies to all ALS services, ref: https://github.com/mojaloop/helm/blob/v13.1.0/.changelog/release-v13.1.0.md#8-upgrade-notes, see section 8.4 for more details.

I don't know if you want to update here, and align to the match the latest updates, or leave that for a separate story?

These changes would also have to be made to ml-api-adapter....so I am leaning to a new story. 

Let me know what you think.

Reply:
separate story